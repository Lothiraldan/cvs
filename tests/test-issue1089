#!/bin/sh

hg init a
cd a
mkdir a
echo a > a/b
hg ci -Am m
hg rm a
hg ci -m m a

mkdir a b
echo a > a/b
hg ci -Am m
hg rm a
cd b
# relative delete
hg ci -m m ../a
