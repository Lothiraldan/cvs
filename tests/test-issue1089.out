adding a/b
removing a/b
adding a/b
removing a/b
